test/PerfHint/arguments1.baseline

```diff
@@ -1,7 +1,7 @@
-PerfHint: Not optimized : Arguments object not optimized due to formals {
+PerfHint: Not optimized : Arguments object not optimized {

```

Comment:
This message is changed again. With formals and arguments object access - This function will not be optimized with StackArgs